bedrock/firefox/templates/firefox/whatsnew/fx54/whatsnew-54.html

```diff
@@ -12,6 +12,12 @@
 
 {% extends "firefox/base-resp.html" %}
 
+{% block experiments %}
+  {% if switch('firefox-experiment-whatsnew-55', variation_locales) %}
```

Comment:
Again a switch here for `whatsnew-55` in the `whatsnew-45` template? I'm sure this is intentional, but just checking?